site/_posts/2016-11-18-Announcing-Euwe-Release-Candidate-2.html.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Euwe Release Candidate 2
+author: chessbyte
+date: 2016-11-21 12:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Release Candidate 2 of ManageIQ Euwe!
+
+Euwe is the next milestone release for ManageIQ cloud and virtualization management platform. With each release, ManageIQ gets more robust and feature complete across providers. Since the Euwe-RC-1 release, we further concentrated on stablization fixes as well as numerous performance improvements prior to the upcoming GA release. A security fix was also added.
+
+Security Fix(es):
+(CVE-2016-7071) It was found that the CloudForms did not properly apply permissions controls to VM IDs passed by users. A remote, authenticated attacker could use this flaw to execute arbitrary VMs on systems managed by CloudForms if they know the ID of the VM.
+This issue was reported by Jeremy Choi (Red Hat).
+
+The following was also added:
+* Auto-tagging based on labels for Containers
+* Add Migrate support to RHEV provider - Support analysis of VMs residing on NFS41 datastores
+
```

Comment:
"Add Migrate support to RHEV provider" and "Support analysis of VMs residing on NFS41 datastores" are different features, please separate them.

The rest LGTM!